src/contract_wrappers/tokens/security_token_wrapper.ts

```diff
@@ -1008,15 +1015,16 @@ export default class SecurityTokenWrapper extends ERC20TokenWrapper {
   };
 
   public addModule: AddModuleInterface = async (params: AddModuleParams) => {
-    const maxCost = params.maxCost === undefined ? BIG_NUMBER_ZERO : params.maxCost;
-    const budget = params.budget === undefined ? BIG_NUMBER_ZERO : params.budget;
+    const maxCost = params.maxCost === undefined ? BIG_NUMBER_ZERO : valueToWei(params.maxCost, await this.decimals());
+    const budget = params.budget === undefined ? BIG_NUMBER_ZERO : valueToWei(params.budget, await this.decimals());
     assert.isETHAddressHex('address', params.address);
     await this.checkOnlyOwner(params.txData);
     await this.checkModuleCostBelowMaxCost(params.address, params.txData, maxCost);
     await this.checkModuleStructAddressIsEmpty(params.address);
     await this.checkUseModuleVerified(params.address);
     let iface: ethers.utils.Interface;
     let data: string;
+    const decimals = await this.decimals();
```

Comment:
You could move this at the beginning to avoid to call it again for maxCost and budget